wandb/sdk/launch/launch.py

```diff
@@ -31,13 +31,13 @@ def resolve_agent_config(
     entity: Optional[str],
     project: Optional[str],
     max_jobs: Optional[int],
-    queues: Optional[List[str]],
+    queues: Tuple[str],
```

Comment:
This should still be optional, as users can still specify queues in the launch config file.